content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -29,7 +29,7 @@ The following methods exist for installing kubectl on Windows:
    Or if you have `curl` installed, use this command:
 
    ```powershell
-   curl -LO "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
+   curl -o kubectl.exe "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
```

Comment:
How about:
```suggestion
   curl -L -o kubectl.exe "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
```
or
```suggestion
   curl --location -o kubectl.exe "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
```

If the problem is that people are using PowerShell pretending to be `curl` (via an alias), then the preceding phrase “if you have `curl` installed” applies and this is a user comprehension error not a docs bug. We could add a comment about PowerShell providing an alias that might mislead. We should aim to retain the follow-redirects behavior; if https://dl.k8s.io/ ever redirects, we'd like to make sure that people follow those redirects, including readers who are actually using the `curl` tool.

Reply:
https://github.com/PowerShell/PowerShell/pull/1901 covers Microsoft's policy on shadowing `curl` with a built-in alias.

Reply:
> Actually, the text is wrong. We could suggest running curl.exe rather than curl?

Came here to say this.
In powershell `curl` is an alias for `Invoke-WebRequest` and doesn't take the same parameters as curl.exe.
curl.exe is standard on all Windows installs so I suggest we just use that everywhere.

Reply:
How about:
```suggestion
   curl.exe -L -o kubectl.exe "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
```
or
```suggestion
   curl.exe --location -o kubectl.exe "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
```